packages/js-recommendations/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import { plugins } from '../../rollup.base.config';
+
+if (!process.env.BUILD) {
+  throw new Error('The `BUILD` environment variable is required to build.');
+}
+
+const output = {
+  umd: {
+    file: 'dist/umd/index.js',
+    format: 'umd',
+    sourcemap: true,
+    name: '@algolia/js-recommendations',
+  },
+  esm: {
+    file: 'dist/esm/index.js',
+    format: 'es',
+    sourcemap: true,
+  },
+};
+
+export default {
+  input: 'src/index.ts',
```

Comment:
We can sync this value with `package.json`:

https://github.com/algolia/react-recommendations/blob/31208f322459ba26a726ff6bff76e6629e6d026a/packages/js-recommendations/package.json#L11

Reply:
Done in https://github.com/algolia/react-recommendations/pull/6/commits/956efe16d5ec7ded275cbac4540230c44dd0904c